src/vcpkg/base/files.cpp

```diff
@@ -1190,50 +1197,50 @@ namespace vcpkg::Files
             failure_point = first->path();
         }
 
-        virtual bool is_directory(const fs::path& path) const override { return fs::stdfs::is_directory(path); }
-        virtual bool is_regular_file(const fs::path& path) const override { return fs::stdfs::is_regular_file(path); }
-        virtual bool is_empty(const fs::path& path) const override { return fs::stdfs::is_empty(path); }
-        virtual bool create_directory(const fs::path& path, std::error_code& ec) override
+        virtual bool is_directory(const path& target) const override { return stdfs::is_directory(target); }
```

Comment:
This isn't introduced by this PR (so don't change it), but it should be noted that this should be `is_directory(this->status(target))`